glommio/src/executor/stall.rs

```diff
@@ -0,0 +1,305 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the MIT/Apache-2.0 License, at your convenience
+//
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2021 Datadog, Inc.
+//
+#![allow(dead_code)]
+
+use nix::sys;
+use std::{
+    fmt,
+    sync::{
+        atomic::{AtomicBool, Ordering},
+        Arc,
+    },
+    thread::JoinHandle,
+    time::{Duration, Instant},
+};
+
+pub struct StallDetection<'a> {
+    executor: usize,
+    task_queue: &'a str,
```

Comment:
the task queue name is fine, but you definitely want the task queue handle in there as well.